llvm/test/Transforms/Attributor/ArgumentPromotion/alignment.ll

```diff
@@ -2,6 +2,11 @@
 ; RUN: opt -aa-pipeline=basic-aa -passes=attributor -attributor-manifest-internal  -attributor-annotate-decl-cs  -S < %s | FileCheck %s --check-prefixes=CHECK,TUNIT
 ; RUN: opt -aa-pipeline=basic-aa -passes=attributor-cgscc -attributor-manifest-internal  -attributor-annotate-decl-cs -S < %s | FileCheck %s --check-prefixes=CHECK,CGSCC
 
+;; Temporarily re-run with RemoveDIs debug-info mode to exercise a
+;; potential crash path.
```

Comment:
I'd argue the "temporarily" isn't necessary - this change is "temporary" in the same way all RemoveDIs check lines are, in that we can get rid of it when RemoveDIs is the default, so I'd just leave it as "Test with RemoveDIs debug-info..."